solution/ui/regulations/eregs-vite/src/components/resources/ResourcesResultsContainer.vue

```diff
@@ -190,22 +179,15 @@ export default {
         },
     },
 
+    inject: ["base"],
+
     data() {
         return {
             activeSortMethod: this.sortMethod,
         };
     },
 
     computed: {
-        filteredContent() {
-            return this.content.map((item) => {
-                const copiedItem = JSON.parse(JSON.stringify(item));
-                copiedItem.locations = item.locations.filter(
-                    (location) => this.partsLastUpdated[location.part]
-                );
-                return copiedItem;
-            });
-        },
```

Comment:
location filtering logic is moved to a single place within the RelatedSections component